data/anaconda_options.txt

```diff
@@ -107,6 +107,9 @@ the password needs to be 6 to 8 characters long (limitation of the VNC protocol)
 xdriver
 Use DRIVER as the X driver to use during installation as well as on the installed system.
 
+xtimeout
+Specify the timeout in seconds for starting X.
```

Comment:
I would add `X server` to be more precise.

Reply:
Fixed.